config/crd/patches/webhook_in_backups.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: backups.k8up.syn.tools
```

Comment:
@ccremer @Kidswiss I'm not sure about these changes. To me, it looks like an artifact from a previous renaming. Do you know more?